### ISSUE TITLE
Maria DB is obsolete now

### ISSUE THREAD

**Issue by laeubi** [ops4j/org.ops4j.pax.jdbc#277]
mariadb already ships with an own DataSourceFactory implementation and could therefor be removed.

**Comment by grgrzybek** [comment]
@laeubi Indeed, proper activator and `org.osgi.service.jdbc.DataSourceFactory` is available in 2.7.7, however I checked latest version 3.1.0 and while it's still JDK8 compatible (and I was thinking that we should keep Pax JDBC 1.5.x JDK8 compatible), the OSGi stuff is gone...

See https://github.com/mariadb-corporation/mariadb-connector-j/commit/e5e95ddfd6ceed7bce757a5b2521c61208e59700#diff-65de1f2f164f96542f8d374b09d176b5be70f7c18edf58384a3b25ec45eca901

**Comment by laeubi** [comment]
Yeah I already asked at the mariadb tracker why they removed that, but probably it could be added back ...

**Comment by grgrzybek** [comment]
@laeubi I've upgraded to MariaDB 3.1.4 and the OSGi stuff is again gone (I mean the Manifest is ok, but no `Bundle-Activator` or any other mean of registering `org.osgi.service.jdbc.DataSourceFactory`.
